app/tasks/doctor.php

```diff
@@ -133,10 +135,17 @@
             }
         }
 
+
         try {
             $mail = $register->get('smtp'); /* @var $mail \PHPMailer\PHPMailer\PHPMailer */
+           
```

Comment:
Also, make sure to run the formatter to fix linting problems.

Reply:
Absolutely, my apologies.